tap/tlstapper/bpf/include/maps.h

```diff
@@ -32,7 +41,8 @@ struct tls_chunk {
     __u32 recorded;
     __u32 fd;
     __u32 flags;
-    __u8 address[16];
+    __u8 fd_address[16];
```

Comment:
I think we can get rid of `fd_address`

Reply:
Fixed 1819f9b134e35d4bf7b160e608beb6383f165632.